webapp/loggedin/surveyViews/taxonomyEdit/components/taxonomyEdit.scss

```diff
@@ -1,36 +1,40 @@
 @import "../../../../style/vars";
 
 .taxonomy-edit {
-  display: grid;
-  grid-template-rows: 5rem 1fr 5rem;
-  align-items: center;
-  height: 100%;
+  height: calc(100% - 50px);
```

Comment:
please use the following layout:
row 1. taxonomy name (with import/export buttons) in the top center (as it was before)
row 2. taxa table
row 3. center bottom the done button (now it's in the left side)